docs/source/using/profiles.adoc

```diff
@@ -31,10 +31,10 @@ There are two ways to create a new profile.
 
 [NOTE]
 ====
-Profile name can only consist of alphanumeric characters and *-* as a separator, for example:
+Profile name can only consist of alphanumeric characters (underscore *_* allowed) and *-* as a separator, for example:
```

Comment:
I think the () placement is a bit off here. Closing ) after allowed. Why not rewrite this a bit:

"Profile names can only consist of alphanumeric characters.  
The use of *_* and *-* as a separator is allowed. For example"